ci/flatpak.sh

```diff
@@ -16,19 +16,20 @@ dnf -y install dnf-plugins-core
 dnf install -y @buildsys-build
 dnf install -y 'dnf-command(builddep)'
 
-# build+install ostree
-dnf builddep -y ostree
+# build+install ostree, and build deps for both, so that our
+# make install overrides the ostree via rpm
+dnf builddep -y ostree flatpak
+dnf install -y flatpak && rpm -e flatpak
+# From the current flatpak builds
+dnf install -y gcc sudo which attr fuse gjs parallel clang libubsan gnome-desktop-testing 'pkgconfig(fuse)' 'pkgconfig(gio-unix-2.0)' 'pkgconfig(gobject-introspection-1.0)' 'pkgconfig(json-glib-1.0)' 'pkgconfig(libarchive)' 'pkgconfig(libelf)' 'pkgconfig(libsoup-2.4)' 'pkgconfig(ostree-1)' 'pkgconfig(polkit-gobject-1)' 'pkgconfig(libseccomp)' 'pkgconfig(xau)' 'bubblewrap' docbook-dtds docbook-style-xsl intltool libattr-devel libcap-devel libdwarf-devel elfutils systemd ostree gpgme-devel /usr/bin/xmlto /usr/bin/xsltproc redhat-rpm-config /usr/bin/update-mime-database /usr/bin/update-desktop-database /usr/bin/gtk-update-icon-cache
```

Comment:
How about we just fetch it from the YAML file after we clone it, e.g. something like:

```
python3 -c 'import yaml; y = list(yaml.load_all(open(".redhat-ci.yml")))[0]; print("\n".join(y["packages"]))' | xargs dnf install -y
```

?